### DISCUSSION TITLE
AAX support: is it possible ?

### DISCUSSION THREAD

**Discussion by fbarton112** [pierreguillot/Camomile#274] [General]
Hey there! I just want to say that I absolutely _LOVE_ this project. At this point, I use my own Camomile generated plugins more often than any other plugins while mixing and doing sound design work. Everyday i am impressed by the stability and reliability of these plugins, they behave just as smoothly as the rest of my plugins. Thank you for making this possible !

Is there any way to offer build support for Pro Tools plugins (AAX format)? I know that they require additional codesigning (from a certified Avid developer), but this should be separate from the actual plugin generation. Is this something that is possible with Camomile? Is there any sort of workaround to do this now, or would it require an update to Camomile itself?

Thank you again! I'm in the processes of putting together a suite of Camomile generated plugins, and i'd be happy to share them here if you're interested. They might be of some use as additional example plugins.

**Comment by pierreguillot** [comment]
Hello,

Sorry for this long delay before any news. 

The most important thing to do I guess is to get the AAX SDK (https://www.avid.com/fr/alliance-partner-program/aax-connectivity-toolkit), compile Camomile for AAX, and test it with ProTools first. So I'll try to do that then I don't know if I can publish the AAX version and how but I'll investigate that.
